### ISSUE TITLE
get ile create? no man...no

### ISSUE THREAD

**Issue by skayikci** [suathh/MovieProject#2]
`$http({
            method: 'GET',
            header:'Access-Control-Allow-Origin',
            url: 'http://localhost:8080/movies/create?name='+$scope.name+'&type='+$scope.type+'&year='+$scope.year+''
        })`

**Comment by suathh** [comment]
got it

**Comment by skayikci** [comment]
have you checked :

[This?](http://stackoverflow.com/questions/6114436/access-control-allow-origin-error-sending-a-jquery-post-to-google-apis)

**Comment by suathh** [comment]
Yes I checked that too but it didn't work for me but I handled it. Today I'll change all HTTP requests with proper ones
